swmm-toolkit/src/swmm/toolkit/__init__.py

```diff
@@ -29,13 +29,16 @@
 
 import os
 import platform
+import sys
 
 
 # Adds directory containing swmm libraries to path
 if platform.system() == "Windows":
     libdir = os.path.join(os.path.dirname(__file__), "../../swmm_toolkit")
 
     if hasattr(os, 'add_dll_directory'):
-        os.add_dll_directory(libdir)
+       if 'conda' in sys.version:
+           os.environ['CONDA_DLL_SEARCH_MODIFICATION_ENABLE']="1"
```

Comment:
Just minor format comment, can you add space between the = and the value?

Reply:
is conda case sensitive in sys.version? would it make sense to make it all lowercase before checking?

Reply:
Good call @jennwuu, I discovered on Stackoverflow that this `conda` check is only valid for python < 3.6 [1]. We no longer support python 3.6. I am updating this `conda` check to be in line with python 3.7 and greater. This patch is adopted from https://stackoverflow.com/questions/47608532/how-to-detect-from-within-python-whether-packages-are-managed-with-conda/47610844#47610844

1 : https://stackoverflow.com/questions/21282363/any-way-to-tell-if-users-python-environment-is-anaconda

Reply:
I added a `conda` test to make sure we are able to install swmm-toolkit via `conda `python. The above addition seems to be working.

Reply:
https://github.com/OpenWaterAnalytics/swmm-python/runs/7355904560?check_suite_focus=true - Test for conda